File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    my_word = ["aya", "ye", "woo", "ma"]
    
    for check in babbling:
        for word in my_word:
            check = check.replace(word, "1", 1)
            print(check)
        if check.isdigit():
            answer += 1
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    let answer = 0;
    const checkWords = ["aya", "ye", "woo", "ma"];
    babbling.forEach((item) => {
        checkWords.forEach((word) => {
            item = item.replace(word, " ")
        })
        if (!item.trim().length) {
            answer += 1;
        }
    })
    return answer;
}